src/components/OutlinedCardList.tsx

```diff
@@ -13,16 +13,19 @@ interface CardProps {
   primary: boolean;
   rightIcon?: JSX.Element;
   columns?: boolean;
+  grayBackground?: boolean;
+  grayText?: boolean;
```

Comment:
if you make the change I described below, you shouldn't need this `greyText`

Reply:
if you look on the figma under smart meter vs no meter/inactive, there is a distinction between when there's gray text and gray background